README.md

```diff
@@ -23,7 +23,7 @@ To start the FastAPI server, navigate to the project's root directory and use th
 
 Note that two .env files are required for the operation of this repo at present time.
 
-1. The first one will be in apps/Learning-Buddy, with the four following fields: IP, SUPABASEURL, SUPABASEKEY_PUBLIC and SUPABASEKEY_ANON. The IP field should be automatically populated with your IPv4 address upon your first time running the frontend with `npm run start:app`.
+1. The first one will be in apps/Learning-Buddy, with the four following fields: IP, SUPABASEURL, SUPABASEKEY_PUBLIC and SUPABASEKEY_SECRET. The IP field should be automatically populated with your IPv4 address upon your first time running the frontend with `npm run start:app`.
```

Comment:
Building .env.example file is better than explaining in the readme.md

Reply:
That's a great idea, Yan! You can add that to our backlog (or create an issue in Github :) ) and we can get to that once you've finished your current task.